### PR TITLE
Timeseries content type validation

### PR DESCRIPTION
**Pull request by pkdash**
### Pull Request Checklist: 
- [x] Positive Test Case Written by Dev

- [x] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [x] Passing Jenkins Build

- [x] Peer Code review and approval

### Positive Test Case
1. Unzip the file below that contains a sqlite file.
2. Use park.hydroshare.org to test. Use the sqlite file to create a TimeSeries resource or TimeSeries aggregation in Composite Resource. This sqlite file has the data that meets the validation requirements that this PR addresses.

[ODM2_multi_site_single_variable.zip](https://github.com/hydroshare/hydroshare/files/2139567/ODM2_multi_site_single_variable.zip)

### CODE REVIEW

**Comment by hydrobuild** [comment]
Build finished.

**Comment by pkdash** [comment]
@hyi Can you review this?

**Comment by dtarb** [comment]
@sblack-usu with Hong away can you please review this.

File: hs_core/testing.py [hunk 1]
```diff
@@ -559,7 +559,7 @@ def timeseries_metadata_extraction(self):
         self.assertEqual(parser.parse(temporal_coverage.value['start']).date(),
                          parser.parse('01/01/2008').date())
         self.assertEqual(parser.parse(temporal_coverage.value['end']).date(),
-                         parser.parse('01/31/2008').date())
+                         parser.parse('01/30/2008').date())
```
Comment (sblack-usu) [inline hs_core/testing.py]: why was this updated?

File: hs_app_timeseries/templates/pages/timeseriesresource.html [hunk 2]
```diff
@@ -37,7 +37,9 @@
                         <table class="custom-table">
                             <tbody>
                                 <tr><th class="text-muted">Code </th><td>{{ site.site_code }}</td></tr>
-                                <tr><th class="text-muted">Name </th><td>{{ site.site_name }}</td></tr>
+                                {% if site.site_name %}
+                                    <tr><th class="text-muted">Name </th><td>{{ site.site_name }}</td></tr>
```
Comment (sblack-usu) [inline hs_app_timeseries/templates/pages/timeseriesresource.html]: do we have guidelines for this type of formatting in a template? I'm ok with this, but it's worth noting the extra indentation is rendered in the html.

File: hs_core/testing.py (hunk 1 above)
Comment (pkdash) [reply hs_core/testing.py]: The temporal coverage data was used to get extracted from a different table of the sqlite file. The issue #2771 suggests to get the same data now from a different table.

Comment (sblack-usu) [reply hs_core/testing.py]: 👍

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by hydrobuild** [comment]
Build finished.

**Comment by pkdash** [comment]
@sblack-usu See if I have addressed your comments.

**Pull request merged**